apps/ivrs/api_views.py

```diff
@@ -49,14 +50,20 @@ def get(self, request):
 
         is_invalid = True
 
-        if not is_user_registered(parameters['telephone']):
+	if parameters['telephone'] == None:
+            message = get_message(parameters, is_user_registered=False)
+
```

Comment:
I am not sure if you want to capture this is as an "unregistered user" error, because it isn't. If you capture this as such, when you generate the SMS report, it will cause confusion to the field team. Because they usually call up the unregistered number and see who they are.

The solution would be to classify this as a completely different sort of error. Maybe classify it as `blank input` or something.